app/src/main/java/org/fossasia/openevent/general/order/OrdersViewHolder.kt

```diff
@@ -29,7 +29,7 @@ class OrdersViewHolder(private val binding: ItemCardOrderBinding) : RecyclerView
         }
 
         itemView.setOnClickListener {
-            listener?.onClick(event.id, order.identifier ?: "", order.id)
+            listener?.onClick(event.id, order.identifier ?: "", order.id, event, order)
```

Comment:
Please change the parameters to a class

Reply:
@iamareebjamal  this OrderClickListener is used in more 3 fragments. If I change here I have to do that at all places. Should I do it?

Reply:
That's the point of refactoring

Reply:
@iamareebjamal ok.all the 5 parameters should be replaced to 1 , right?

Reply:
First of all, why are you transferring entire event and order. By that logic, there is no requirement for transferring first 3 parameters